code/frameworks/react-vite/src/plugins/react-docgen.ts

```diff
@@ -36,14 +36,21 @@ export function reactDocgen({
     name: 'storybook:react-docgen-plugin',
     enforce: 'pre',
     async transform(src: string, id: string) {
-      const relPath = path.relative(cwd, id);
-      if (!filter(relPath)) return;
+      if (!filter(path.relative(cwd, id))) {
+        return;
+      }
 
       try {
         const docgenResults = parse(src, {
           resolver: defaultResolver,
           handlers,
-          importer: defaultImporter,
+          importer: makeFsImporter((filename, basedir) => {
+            if (!filter(path.relative(cwd, join(basedir, filename)))) {
+              throw new ReactDocgenResolveError(filename);
+            }
```

Comment:
So this filtering might be filtering too much... causing this:
https://www.chromatic.com/test?appId=630511d655df72125520f051&id=65e067dddcd6da9ad2e2657e

Going to investigate this further.

Reply:
So I simplified the filtering, but it's not helping with the regression.
Need to investigate further.